internal/cmd/postgresflex/instance/clone/clone_test.go

```diff
@@ -0,0 +1,325 @@
+		{
+			description: "no values",
+			argValues:   []string{},
+			flagValues:  map[string]string{},
+			isValid:     false,
+		},
+		{
+			description: "no arg values",
+			argValues:   []string{},
+			flagValues:  fixtureRequiredFlagValues(),
+			isValid:     false,
+		},
+		{
+			description: "no flag values",
+			argValues:   fixtureArgValues(),
+			flagValues:  map[string]string{},
+			isValid:     false,
+		},
+		{
+			description: "all values with storage class",
+			argValues:   fixtureArgValues(),
+			flagValues: fixtureStandardFlagValues(func(flagValues map[string]string) {
+				delete(flagValues, storageSizeFlag)
+				flagValues[storageClassFlag] = "premium-perf4-stackit"
+			}),
+			isValid: true,
+			expectedModel: fixtureStandardInputModel(func(model *inputModel) {
+				model.StorageSize = nil
+				model.StorageClass = utils.Ptr("premium-perf4-stackit")
+			}),
+		},
+		{
+			description: "all values with storage size",
+			argValues:   fixtureArgValues(),
+			flagValues: fixtureStandardFlagValues(func(flagValues map[string]string) {
+				delete(flagValues, storageClassFlag)
+				flagValues[storageSizeFlag] = "2"
+			}),
+			isValid: true,
+			expectedModel: fixtureStandardInputModel(func(model *inputModel) {
+				model.StorageClass = nil
+				model.StorageSize = utils.Ptr(int64(2))
+			}),
+		},
+		{
+			description: "project id missing",
+			argValues:   fixtureArgValues(),
+			flagValues: fixtureRequiredFlagValues(func(flagValues map[string]string) {
+				delete(flagValues, projectIdFlag)
+			}),
+			isValid: false,
+		},
+		{
+			description: "project id invalid 1",
+			argValues:   fixtureArgValues(),
+			flagValues: fixtureRequiredFlagValues(func(flagValues map[string]string) {
+				flagValues[projectIdFlag] = ""
+			}),
+			isValid: false,
+		},
+		{
+			description: "project id invalid 2",
+			argValues:   fixtureArgValues(),
+			flagValues: fixtureRequiredFlagValues(func(flagValues map[string]string) {
+				flagValues[projectIdFlag] = "invalid-uuid"
+			}),
+			isValid: false,
+		},
+		{
+			description: "instance id invalid 1",
+			argValues:   []string{""},
+			flagValues:  fixtureRequiredFlagValues(),
+			isValid:     false,
+		},
+		{
+			description: "instance id invalid 2",
+			argValues:   []string{"invalid-uuid"},
+			flagValues:  fixtureRequiredFlagValues(),
+			isValid:     false,
+		},
+		{
+			description: "recovery timestamp is missing",
+			flagValues: fixtureRequiredFlagValues(func(flagValues map[string]string) {
+				delete(flagValues, recoveryTimestampFlag)
+			}),
+			isValid: false,
+		},
+		{
+			description: "recovery timestamp is empty",
+			flagValues: fixtureRequiredFlagValues(func(flagValues map[string]string) {
+				flagValues[recoveryTimestampFlag] = ""
+			}),
+			isValid: false,
+		},
+		{
+			description: "recovery timestamp is invalid",
+			flagValues: fixtureRequiredFlagValues(func(flagValues map[string]string) {
+				flagValues[recoveryTimestampFlag] = "test"
+			}),
+			isValid: false,
```

Comment:
Because this test seems to be passing, meaning the invalid timestamp is catched somewhere right?

Reply:
timestamp validation has been added